app/views/courses/index.html.erb

```diff
@@ -0,0 +1,85 @@
+<%= content_for :page_title, "Courses" %>
+
+<nav class="govuk-breadcrumbs">
+  <ol class="govuk-breadcrumbs__list">
+    <% if @has_multiple_providers then %>
+      <li class="govuk-breadcrumbs__list-item">
+        <%= link_to "Organisations", providers_path, class: "govuk-breadcrumbs__link" %>
+      </li>
+    <% end %>
+    <li class="govuk-breadcrumbs__list-item">
+      <%= link_to @provider[:institution_name], provider_path(code: @provider[:institution_code]), class: "govuk-breadcrumbs__link" %>
+    </li>
+    <li class="govuk-breadcrumbs__list-item" aria-current="page">
+      Courses
+    </li>
+  </ol>
+</nav>
+
+<main role="main" class="govuk-main-wrapper" id="main-content">
+  <h1 class="govuk-heading-xl">Courses</h1>
+
+  <p class="govuk-body">Use this section to:</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <li>write about each course</li>
+    <li>preview and publish courses</li>
+    <li>copy content between courses</li>
+  </ul>
+
+  <h2 class="govuk-heading-m" style="border: 3px solid red">
```

Comment:
Do we want to be using `style` attributes?

Reply:
Yes, because they are temporary and will be removed. I just want to merge this because it's not accessible to the user and to prevent it from getting bigger.